src/main/java/org/jabref/gui/openoffice/ManageCitationsDialogView.java

```diff
@@ -75,14 +74,13 @@ private void initialize() throws NoSuchElementException, WrappedTargetException,
             cell.getRowValue().setExtraInfo(cell.getNewValue());
         });
         extraInfo.setCellFactory(TextFieldTableCell.forTableColumn());
-
     }
 
     private Node getText(String citationContext) {
 
         String inBetween = StringUtil.substringBetween(citationContext, HTML_BOLD_START_TAG, HTML_BOLD_END_TAG);
         String start = citationContext.substring(0, citationContext.indexOf(HTML_BOLD_START_TAG));
-        String end = citationContext.substring(citationContext.lastIndexOf(HTML_BOLD_END_TAG) + HTML_BOLD_END_TAG.length(), citationContext.length());
+        String end = citationContext.substring(citationContext.lastIndexOf(HTML_BOLD_END_TAG) + HTML_BOLD_END_TAG.length());
```

Comment:
Why did you remove the last parameter?

Reply:
That was done automatically by IntellJ. The substring method always takes the remaining string if only one index is passed. https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#substring(int)